lib/utils/thingtalk/describe.ts

```diff
@@ -174,7 +174,7 @@ export class Describer {
         let base;
 
         if (date === null) {
-            base = this._const(this._("now"));
+            base = this._const(this._("today"));
```

Comment:
This is not correct. `new Date()` on its own (which will be represented by DateValue with `null` date field) is "now", not "today".

Instead, we should special case just `set_time(new Date, ...)` to be "XX:XX today" (or "today at XX:XX").